src/responses/page-elements.ts

```diff
@@ -73,6 +73,10 @@ export interface ForumPost {
 }
 
 export class SearchReview extends Review {
+  get reviewer_account_status(): string | undefined {
```

Comment:
Q: Would it make sense to restrict this to a type union of the expected statuses?